Formula/aws-rotate-key.rb

```diff
@@ -0,0 +1,19 @@
+class AwsRotateKey < Formula
+  desc "Easily rotate your AWS access key. 🔑"
+  homepage "https://github.com/stefansundin/aws-rotate-key"
+  url "https://github.com/stefansundin/aws-rotate-key/archive/v1.0.7.tar.gz"
+  sha256 "9dadb689583dc4d8869a346c2e7f12201e1fe65d5bf1d64eb09b69f65518bc44"
+  license "MIT"
+  head "https://github.com/stefansundin/aws-rotate-key.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+  
+  test do
+    assert_match(%r{/^Usage of #{bin}/aws-rotate-key:/}, shell_output("#{bin}/aws-rotate-key --help 2>&1"))
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
sounds good. Let me see if I can backfill or figure out with the developer if he can add those

Reply:
Well, you need to add them to the formula. They don't need to be upstream.

Reply:
sounds good.